### ISSUE TITLE
Not executable - missing data

### ISSUE THREAD

**Issue by brandondube** [kian1377/Roman-CGI-POPPY#2]
Hi Kian,

I'm working on a manuscript for JATIS on modeling the Roman Coronagraph Instrument at kilohertz in real-time.  In it, I compare the runtime of my model to the POPPY model (sourced from this repo), and the PROPER model, which I've obtained directly from John Krist.

To keep things apples to apples, I want to run all three models on exactly the same machine and environment.  Unfortunately, your repo does not contain the `roman-cgi-poppy-data` directory, and when I look, it is John's distribution but with the folder structure changed (I guess), since the code errors looking for `roman-cgi-poppy-data/optics/F825/pupil_SPC-20181220_1k_rotated.fits'`.

Is there a public source for the `roman-cgi-poppy-data` folder that you could direct me to?

Thanks,
-Brandon

**Comment by kian1377** [comment]
Hello,

So I want to start off by saying that this repo was originally made for the
Phase-B models which are now outdated since phase-c was released last
summer. As for the data, it is mostly just a changed folder structure for
all the data files included for the PROPER models since we originally
intended to add these Roman models to WebbPSF, so we wanted to follow its
directory structure more closely. However, the data files are
slightly altered so that POPPY can automatically read in important
information from the headers of the FITS files.

The data directory I used can be found on BOX at this link:
https://arizona.box.com/s/9cquzldre5ru2497a70omstxb1o83l5s
The directiry on BOX is capitalized differently than the one I used on my
machine so just change that to run everything.

Regards,
Kian Milani
